sites/fast-website/src/app/components/fast-frame/fast-frame.styles.ts

```diff
@@ -20,7 +20,8 @@ export const FastFrameStyles = css`
     }
 
     :host {
-        --gutter: 20;
+				--gutter: 20;
```

Comment:
Is prettier adding all this whitespace?

Reply:
looks like tabs instead of spaces. Is it possible that since this is a template literal, Prettier isn't picking it up?

Reply:
It must be.  I know I didn't add it intentionally.  Let me see if I can do something to fix it.

Reply:
It was tabs instead of spaces for some reason, and running prettier did not catch it.  Got it fixed manually so should be all good now.